x-pack/plugins/apm/jest_setup.js

```diff
@@ -4,12 +4,3 @@
  * 2.0; you may not use this file except in compliance with the Elastic License
```

Comment:
Can this be deleted?

Reply:
It was only used for spying on `console.error` so I see no reason for having it any longer.

Reply:
I agree, I mean can you delete the whole file?

Reply:
Oh, I thought that's what I did. Will delete.

Reply:
Done.